proposals/description/tip_vebalotc.ts

```diff
@@ -0,0 +1,175 @@
+      method: 'exitPool(address)',
+      arguments: (addresses) => [addresses.daiHoldingPCVDeposit],
+      description: 'Move all DAI and WETH to daiHoldingPCVDeposit'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.daiHoldingPCVDeposit, // pcvDeposit
+        addresses.weth, // token
+        addresses.tribalCouncilSafe, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Move all WETH from daiHoldingPCVDeposit to the TC Multisig'
+    },
+
+    // 5. Cleanup FEI/TRIBE on TC Timelock from the Rari Infra team clawback
+    {
+      target: 'tribe',
+      values: '0',
+      method: 'transferFrom(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribalCouncilTimelock, addresses.core, '2733170474316903966022879'],
+      description: 'Move all TRIBE from TC Timelock to DAO Treasury'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'transferFrom(address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.tribalCouncilTimelock,
+        addresses.feiDAOTimelock,
+        '2733169815107120096987175'
+      ],
+      description: 'Move all FEI from TC Timelock to DAO Timelock'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'burn(uint256)',
+      arguments: (addresses) => ['2733169815107120096987175'],
+      description: 'Burn FEI from the DAO Timelock'
+    },
+
+    // Finally, CR Oracle updates
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'removeDeposits(address[])',
+      arguments: (addresses) => [
+        [
+          addresses.balancerLensVeBalBal,
+          addresses.balancerLensVeBalWeth,
+          addresses.balancerGaugeStaker,
+          addresses.balancerDepositBalWeth,
+          addresses.aaveFeiPCVDepositWrapper,
+          addresses.wethHoldingPCVDeposit,
+          addresses.lusdToDaiLensLusd,
+          addresses.lusdToDaiLensDai,
+          addresses.ethToDaiLensDai,
+          addresses.ethToDaiLensEth,
+          addresses.tribalCouncilTimelockFeiLens
+        ]
+      ],
+      description: 'Remove deprecated/empty smart contracts from CR Oracle'
+    }
+  ],
+  description: `
+  TIP-121b: veBAL OTC
+
+  This proposal sets up the protocol to be able to transfer its 112k veBAL to Aave Companies for OTC trade.
+
+  This proposal also includes several cleanup items to wrap up previous proposals.
+
+  The full list of actions of this proposal is :
+  - 1. Handle veBAL OTC
+  - 2. Move dust Balancer-related funds to TC Multisig for selling
+  - 3. Cleanup LUSD->DAI Auction contracts (send dust to TC multisig for selling)
+  - 4. Cleanup WETH->DAI Auction contracts (send dust to TC multisig for selling)
+  - 5. Cleanup FEI/TRIBE on TC Timelock from the Rari Infra team clawback (send TRIBE to DAO treasury, burn FEI)
```

Comment:
These might need removal if another proposal passes onchain before & takes care of these cleanup items.

Reply:
Imo the cleanup items should be a separate PR

Reply:
ok

Reply:
I've split into a separate proposal file, we'll want to consolidate it with the next vote that goes onchain